lib/src/runner/configuration/args.dart

```diff
@@ -65,7 +65,7 @@ final ArgParser _parser = (() {
       abbr: 'p',
       help: 'The platform(s) on which to run the tests.\n'
           '[vm (default), '
-          '${allPlatforms.map((platform) => platform.identifier).join(", ")}]',
+          '${allRuntimes.map((runtime) => runtime.identifier).join(", ")}]',
```

Comment:
Should we add some internal comments here stating platforms map to runtimes due to historical reasons?